plugins/pcl/filters/PCLBlock.cpp

```diff
@@ -78,7 +78,7 @@ PointBufferSet PCLBlock::run(PointBufferPtr input)
 
     BOX3D const& buffer_bounds = input->calculateBounds();
 
-    // convert PointBuffer to PointNormal
+    // convert PointView to PointNormal
     typedef pcl::PointCloud<pcl::PointXYZ> Cloud;
     Cloud::Ptr cloud(new Cloud);
     pclsupport::PDALtoPCD(*input, *cloud, buffer_bounds);
```

Comment:
`Error	473	error C2664: 'void pdal::pclsupport::PDALtoPCD<T>(pdal::PointViewPtr,CloudT &,const pdal::BOX3D &)' : cannot convert parameter 1 from 'pdal::PointView' to 'pdal::PointViewPtr'`